tests/x11/toolkits/prepare.pm

```diff
@@ -26,6 +26,8 @@ sub run {
             assert_script_run 'SUSEConnect -p sle-module-development-tools/${VERSION_ID}/${CPU}';
         }
         else {
+            # for some reason we don't register SLE12 systems in openqa by default
```

Comment:
It's not "for some reason" but for "historical reasons" ;)

We started with the SLE12 automated tests before there was proxy-SCC. But we have specific scenarios that still do the registration. I thought I explained in IRC in detail already.

And since then we have simply never switched it around as this would be way more work than you might expect, to crosscheck everything and such.